packages/wasm/crate/src/tree.rs

```diff
@@ -0,0 +1,18 @@
+use penumbra_proto::DomainType;
+use penumbra_sct::epoch::Epoch;
+use penumbra_tct::Position;
+use wasm_bindgen::prelude::wasm_bindgen;
+
+use crate::error::WasmResult;
+
+#[wasm_bindgen]
+pub fn sct_position(block_height: u64, epoch_bytes: &[u8]) -> WasmResult<u64> {
+    let epoch = Epoch::decode(epoch_bytes)?;
+    let epoch_index = u16::try_from(epoch.index)?;
+    // The block index is determined by looking at how many blocks have elapsed since
+    // the start of the epoch.
+    let block_index = u16::try_from(block_height - epoch.start_height)?;
+
+    let position = Position::from((epoch_index, block_index, 0u16));
+    Ok(position.into())
```

Comment:
Seems like we could have done it without wasm?

Reply:
The tree position is serialized using bitwise operators: https://github.com/penumbra-zone/penumbra/blob/8d4dc0330a8bf86a4334987a02f0fa04ac40baa0/crates/crypto/tct/src/index.rs#L228-L238.

I thought about implementing a version of this in typescript, but I figured 1) it's easier to use what they created and 2) we'll be attempting to migrate this to wasm soon anyway.

Reply:
Makes sense